src/imagery/i.eodag/i.eodag.py

```diff
@@ -667,6 +760,8 @@ def main():
         # TODO: Add timeout and wait parameters for downloading offline products...
         # https://eodag.readthedocs.io/en/stable/getting_started_guide/product_storage_status.html
         dag.download_all(search_result)
+        if options["footprints"]:
```

Comment:
I have a clear idea but maybe people could be interested to download the footprints without download all the data. Please look at i.sentinel.download https://github.com/OSGeo/grass-addons/blob/grass8/src/imagery/i.sentinel/i.sentinel.download/i.sentinel.download.py#L1282